internal/env/env.go

```diff
@@ -22,6 +20,8 @@ const (
 	TemplateValidatorImageKey = "VALIDATOR_IMAGE"
 	VmConsoleProxyImageKey    = "VM_CONSOLE_PROXY_IMAGE"
 
```

Comment:
Drop newline between?

Reply:
The newline separates exported and private constants.